sched/group/group_create.c

```diff
@@ -119,9 +121,23 @@ int group_initialize(FAR struct task_tcb_s *tcb, uint8_t ttype)
 
   DEBUGASSERT(tcb && !tcb->cmn.group);
 
+  ttype &= TCB_FLAG_TTYPE_MASK;
+
+  if (ttype == TCB_FLAG_TTYPE_KERNEL && g_kthread_group.tg_info)
```

Comment:
```
if (ttype == TCB_FLAG_TTYPE_KERNEL)
  {
    group = &g_kthread_group;
    tcb->cmn.group = group;
    if (group->tg_info)
     {
       return OK;
     }
  }
else
  {
    group =  &tcb->group;
  }
```

Reply:
okay will do later, the debugging size dumps can be added locally if needed.

Reply:
yes.